go/vt/vtgate/planbuilder/horizon_planning.go

```diff
@@ -457,10 +456,7 @@ func addColumnsToOA(
 			o := groupings[count]
 			count++
 			a := aggregationExprs[offset]
-			collID := collations.Unknown
-			if aggr, ok := a.Func.(sqlparser.AggrFunc); ok {
-				collID = ctx.SemTable.CollationForExpr(aggr.GetArg())
-			}
+			collID := ctx.SemTable.CollationForExpr(a.Func.GetArg())
```

Comment:
Linter mentioned that this was a useless type check since it's already an `AggrFunc` so I cleaned that up.